### ISSUE TITLE
Question regarding boxes

### ISSUE THREAD

**Issue by AakashKumarNain** [tianzhi0549/FCOS#147]
1. Do you normalize bounding boxes in the dataloader? Also, what is the order of boxe coordinates? Is it `xmin, xmax, ymin, ymax` or `ymin, xmin, ymax, xmax`?

2. What is the order of the predicted bounding box coordinates?

**Comment by tianzhi0549** [comment]
@AakashKumarNain No. We do not normalize bounding boxes in dataloader. The normalization takes place here https://github.com/tianzhi0549/FCOS/blob/4697f876b5aeead5f60423e0d04ea7e3e1282790/fcos_core/modeling/rpn/fcos/loss.py#L140.

The order is left, top, right and bottom.
